### PR TITLE
Layout fixes

### PR DESCRIPTION
**Pull request by direc85**
Hello,

I notices some layout issues in AppListDelegate especially when system font size is set to Large or Huge, so I redesigned the whole thing (but it looks almost idencal) and added extra padding for landscape orientation, I think it looks really nice :) I tested it with all the devices and orientations available in the SDK, plus Xperia X with all font sizes, too, and it works perfectly for me.

I also fixed comments clipping into page header, fine-tuned application page layout and updated (completed) Finnish translation.

Thanks!

### CODE REVIEW

**Comment by mentaljam** [comment]
Hi,
Thank you for your contribution. Sorry, I haven't time to check your PRs now.

**Comment by direc85** [comment]
No problem, when you have time.

**Comment by direc85** [comment]
Thanks! I'll check the code and test the layout later this week (I hope).

File: qml/components/AppListDelegate.qml [hunk 1]
```diff
@@ -27,104 +35,109 @@ ListItem {
...
+
+        // Bookmark star on the right
+        BookmarkButton {
+            id: bookmarkStar
+            anchors.top: parent.top
+            anchors.right: parent.right
+            anchors.bottom: userNameLabel.top
+            width: Theme.iconSizeLauncher
         }
-        appId: model.appId
```
Comment (mentaljam) [inline qml/components/AppListDelegate.qml]: Regression, fixed in layout_fixes

File: qml/components/AppListDelegate.qml [hunk 2]
```diff
@@ -27,104 +35,109 @@ ListItem {
...
+            ratingCount: model.ratingCount
+            rating: model.rating
+        }
 
-                Label {
-                    id: userNameLabel
-                    anchors.verticalCenter: parent.verticalCenter
-                    width: column.width - ratingBox.width - statusLoader.width - parent.spacing * 2
-                    horizontalAlignment: Qt.AlignRight
-                    truncationMode: TruncationMode.Fade
```
Comment (mentaljam) [inline qml/components/AppListDelegate.qml]: Regression, fixed in layout_fixes

**Comment by direc85** [comment]
I had a quick look, looks good to me! I tested it on Jolla 1 and Xperia X, with all three font sizes in portrait and in landscape.

**Comment by mentaljam** [comment]
Did you tested with the latest commits? I've force pushed some changes to layout_fixes so you may need to rebase your branch.

**Comment by direc85** [comment]
Yeah, I made sure I was using this branch.

**Pull request closed without merging**